packages/design-system/src/components/Accordion/index.tsx

```diff
@@ -33,18 +33,21 @@ const AccordionComponent: React.FC<AccordionProps> = ({
                   iconPosition={iconPosition}
                   icon={
                     <ChevronDownSvg
-                      className={classNames("w-accordion-widthChevron transform transition", {
-                        "rotate-0": isExpanded,
-                        "-rotate-90": !isExpanded,
-                      })}
+                      className={classNames(
+                        "w-accordion-chevron transform transition",
```

Comment:
The same like with `minus` token
```suggestion
                        "w-accordion-wChevron transform transition",
```

Reply:
same